apps/minifront/src/components/header/menu/provider.tsx

```diff
@@ -60,7 +59,11 @@ export const ProviderMenu = () => {
             </NavigationMenu.Link>
           </NavigationMenu.Item>
           <NavigationMenu.Item className={cn(...itemStyle)}>
-            <NavigationMenu.Link hidden className={cn(...linkStyle)} onSelect={disconnect}>
+            <NavigationMenu.Link
+              hidden
+              className={cn(...linkStyle)}
+              onSelect={() => void penumbra.disconnect()}
```

Comment:
question: is the disconnect button intentionally hidden?

also, calling the disconnect should probably redirect user to the connection page. Right now it simply displays an error in the block syncing

Reply:
rebase error, will fix